src/opera/api/cli.py

```diff
@@ -18,8 +17,8 @@ def main():
         server = "tornado"
 
     app = connexion.App(__name__, specification_dir="./openapi/openapi/", server=server, options=dict(
-        serve_spec=False,
-        swagger_ui=False
+        serve_spec=True,
```

Comment:
Exposing Swagger UI and API specification is usually not recommended for production. A suggestion is to enable it only in DEBUG mode, like this:
```python
serve_spec=DEBUG,
swagger_ui=DEBUG
```

Reply:
Thanks for the suggestion - I partially agree about hiding the SwaggerUI and I have incorporated this change.

I will just quote this from FastAPI [page](https://fastapi.tiangolo.com/pt/advanced/conditional-openapi/):

> Hiding your documentation user interfaces in production shouldn't be the way to protect your API.
> 
> That doesn't add any extra security to your API, the path operations will still be available where they are.
> 
> If there's a security flaw in your code, it will still exist.
> 
> Hiding the documentation just makes it more difficult to understand how to interact with your API, and could make it more difficult for you to debug it in production.